src/mainboard/protectli/vault_adl_n/variants/vp2440/overridetree.cb

```diff
@@ -24,21 +24,23 @@ chip soc/intel/alderlake
 		end
 
 		device ref xhci on
-			register "usb2_ports[0]" = "USB2_PORT_EMPTY"
-			register "usb2_ports[5]" = "USB2_PORT_SHORT(OC_SKIP)" # 5G/LTE
-			register "usb3_ports[1]" = "USB3_PORT_EMPTY"
+			device ref xhci_root_hub on
```

Comment:
Don't you need `chip drivers/usb/acpi` as a parent of this node?

Reply:
you're right, https://github.com/Dasharo/coreboot/pull/678/commits/2b165dd1c41b3fcfc8014abb4d2a8e4bf119279b